versions.json

```
@@ -1,38 +1,30 @@
 [
-  {
-    "version": "6.9.1",
-    "static_tag": "stable"
-  },
-  {
-    "version": "6.9",
-    "static_tag": "mainline"
-  },
-  {
-    "version": "6.8.10",
-    "static_tag": ""
-  },
   {
     "version": "6.6.31",
```

nit: things are moving fast
```suggestion
    "version": "6.6.51",
```